File: lib/src/main/java/one/hyro/spark/lib/api/dto/player/PlayerData.java
package one.hyro.spark.lib.api.dto.player;

import one.hyro.spark.lib.common.Rank;

import java.util.Date;
import java.util.UUID;

public record PlayerData(
    UUID uuid,
    Rank rank,
    Date rank_expiration,
    Date first_joined,
    Date last_seen
) {}

File: lib/src/main/kotlin/api/dto/player/PlayerData.kt
package one.hyro.api.dto.player

import one.hyro.common.Rank
import java.util.Date
import java.util.UUID

data class PlayerData(
    val uuid: UUID,
    val rank: Rank,
    val firstJoined: Date,
    val lastSeen: Date?
)
